source/clear-linux/tutorials/telemetry-backend/telemetry-backend.rst

```diff
@@ -462,14 +462,17 @@ it.
 Additional resources
 ********************
 
-https://clearlinux.org/features/telemetry
-
-https://github.com/clearlinux/telemetrics-client
-
-https://github.com/clearlinux/telemetrics-backend
+* `Telemetry feature description`_
+* :ref:`Telemetry architecture<telemetry-about>`
+* :ref:`Enable and disable telemetry in Clear Linux<telemetry-enable>`
+* https://github.com/clearlinux/telemetrics-client
+* https://github.com/clearlinux/telemetrics-backend
 
 .. _`Clear Linux telemetry backend server overview`:
    https://github.com/clearlinux/telemetrics-backend
 
 .. _`Intel privacy policies`:
    https://www.intel.com/content/www/us/en/privacy/intel-privacy-notice.html
+
+.. _`Telemetry feature description:
+  `https://clearlinux.org/features/telemetry
```

Comment:
AR Mary fix these 2 lines so it looks like this:
.. _`Telemetry feature description`:https://clearlinux.org/features/telemetry